recipes/multidict/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "multidict" %}
+{% set version = "2.1.4" %}
+{% set sha256 = "a77aa8c9f68846c3b5db43ff8ed2a7a884dbe845d01f55113a3fba78518c4cd7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  source_files:
+    - tests
+  requires:
+    - pytest
+  commands:
+    - py.test tests
+
+about:
+  home: http://github.com/aio-libs/multidict
+  license: Apache 2.0
+  license_file: LICENSE
+  summary: 'multidict implementation'
+
+  description: |
+    Multidicts are useful for working with HTTP headers, URL query args etc.
+  doc_url: http://simplejson.readthedocs.io/
+  dev_url: http://github.com/aio-libs/multidict
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
+    - HomerSimpson
+    - LukeSkywalker
```

Comment:
I will pretend that I was just trying to see if you were paying attention.

Thanks for catching that!